src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -285,18 +373,18 @@ void BaseCouplingScheme::addComputedTime(
     double timeToAdd)
 {
   PRECICE_TRACE(timeToAdd, _time);
-  PRECICE_ASSERT(isCouplingOngoing(), "Invalid call of addComputedTime() after simulation end!");
+  PRECICE_ASSERT(isCouplingOngoing(), "Invalid call of addComputedTime() after simulation end.");
 
   // add time interval that has been computed in the solver to get the correct time remainder
   _computedTimeWindowPart += timeToAdd;
   _time += timeToAdd;
 
   // Check validness
   bool valid = math::greaterEquals(getThisTimeWindowRemainder(), 0.0, _eps);
-  PRECICE_CHECK(valid, "The computed timestep length of "
-                           << timeToAdd << " exceeds the maximum timestep limit of "
+  PRECICE_CHECK(valid, "The computed time step length of "
```

Comment:
For preCICE it's always "timestep", not "time step". Only in the QNWI paper we used "time step" :see_no_evil:

Reply:
Fixed in 7b0396f495609c1074b9978ddb2c0d3f0bf56f7d